### PR TITLE
[docs] refactor and simplify global styles

### PR DESCRIPTION
**Pull request by Simek**
# Why

Probably second to last part of Emotion styling removal efforts. This time the global styles has been a target.

# How

Replace Emotion based global styles with single CSS file utilizing already existing Tailwind classes.

There should be no visual differences* between currently deployed site and this PR.

> *There were also few small tweaks and fixes for the existing styles, which might appear when testing on mobile, but nothing major to mention.

# Test Plan

The changes have been reviewed by running docs app locally. All lint checks and tests are passing.

# Preview

Screenshot 2024-10-26 at 20 13 12
Screenshot 2024-10-26 at 20 14 00
Screenshot 2024-10-26 at 20 14 38

### CODE REVIEW

File: docs/styles/global.css [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+  .token.atrule,
+  .token.attr-value,
+  .token.function {
+    @apply text-palette-pink10;
+  }
+}
+
+/* Misc */
+
+img[src*="https://placehold.it/15"] {
```
Comment (amandeepmittal) [inline docs/styles/global.css:338]: Is this placeholder value required?

**Review by amandeepmittal — CHANGES_REQUESTED** [review]
It seems the tooltip for light theme is not showing up:

CleanShot 2024-10-29 at 01 40 54@2x

vs.

CleanShot 2024-10-29 at 01 41 02@2x

**Comment by Simek** [comment]
Thanks for catching that! Fix is up:

Screenshot 2024-10-29 at 11 14 12

File: docs/styles/global.css (hunk 1 above)
Comment (Simek) [reply docs/styles/global.css]: It looks like it's unused, removing!

**Review by amandeepmittal — APPROVED** [review]
Thank you!

**Pull request merged**